.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+### Description
+
+
+#### Related issues
+
+<!--
+Please list related issues and discussion by using the following syntax:
+
+- Relates to #49
+  (to reference issues in the Objection.js repository)
+- Relates to https://github.com/tgriesser/knex/issues/100
+  (to reference issues in a related repository)
+-->
+
+- Relates to <link_to_referenced_issue>
+
+### Checklist
+
+<!--
+- Please mark your choice with an "x" (i.e. [x], see
+https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments)
+- PR's without test coverage will be closed.
+-->
+
+- [ ] New tests added or existing tests modified to cover all changes
+- [ ] Code conforms with the ESLint and Prettier rules (`yarn eslint` passes
```

Comment:
I think we should use `npm` examples instead of `yarn` here.

Reply:
Oops yes, that's just the habit taking over : )